status.go

```diff
@@ -29,3 +33,22 @@ func StatusText(status int) string {
 
 	return UnknownString
 }
+
+// StatusText returns a state corresponding to its
+// common string representation
+func StatusInt(status string) int {
+	status = strings.ToUpper(status)
+
+	switch status {
+	case OKString, "0":
+		return OK
+	case WarningString, "1":
+		return Warning
+	case CriticalString, "2":
+		return Critical
+	case UnknownString, "3":
+		return Unknown
+	default:
+		return Unknown
```

Comment:
I do understand the wish for consistency, but IMHO there is a clear error case here and I feel uncomfortable with the function handling this the way it is now.

I think I would be rather (unpleasantly) surprised to find out, that this function transforms "Critiacl" (a typo of "Critical") to `Unknown` without giving me a way to verify it.

And yes, the function is rather trivial and I might be heavily prone to overengineering.